### ISSUE TITLE
Missing documentation on signalOptions settings UI

### ISSUE THREAD

**Issue by chrmarti** [microsoft/vscode#213670]
Testing #213344

There no documentation on signalOptions:
![Image](https://github.com/microsoft/vscode/assets/9205389/f173c9b1-1bc8-4c33-80e8-70e48b3b7a47)

There is some documentation on the leaf properties in the JSON, but it is unclear what `"general"` refers to.

**Comment by connor4312** [comment]
I think the request was to have an overall description for the setting, as right now there's nothing until a user starts to complete properties in the settings.json

<img width="636" alt="image" src="https://github.com/microsoft/vscode/assets/2230985/62bde016-0801-47a1-9f68-d2f60ac616b2">
